### PR TITLE
feat(widget-chat): Typing indicator

### PR DESCRIPTION
**Pull request by adamweeks**

### CODE REVIEW

File: packages/widget-chat/src/actions/indicators.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+export const ADD_TYPING_INDICATOR = `ADD_TYPING_INDICATOR`;
+export function addTyping(id) {
```
Comment (bzang) [inline packages/widget-chat/src/actions/indicators.js]: nitpick: use more specific var name (e.g. `userId`)?

File: packages/widget-chat/src/components/typing-indicator/styles.css [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.background {
+  width: 30px;
+  height: 30px;
+  padding: 0;
+  text-align: center;
+  cursor: default;
+  background-color: #696969;
+  border: none;
+  border-radius: 15px;
+}
```
Comment (bzang) [inline packages/widget-chat/src/components/typing-indicator/styles.css]: Does the current web client UI animate the ellipses? If so, we may want to get that in now.

File: packages/widget-chat/src/containers/scrolling-activity/index.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    </div>
+  );
+}
+
+ScrollingActivity.propTypes = {
+  activities: PropTypes.array,
+  currentUserId: PropTypes.string,
+  isTyping: PropTypes.bool,
+  onActivityDelete: PropTypes.func.isRequired,
+  onScrollToBottom: PropTypes.func
```
Comment (bzang) [inline packages/widget-chat/src/containers/scrolling-activity/index.js]: Is this an extra prop that isn't being used?

File: packages/widget-chat/src/actions/indicators.js (hunk 1 above)
Comment (adamweeks) [inline packages/widget-chat/src/actions/indicators.js]: Fixed.

**Comment by adamweeks** [comment]
@bzang ready for round two

**Pull request merged**